app/utils/errors.js

```diff
@@ -26,19 +26,25 @@ const errorProps = {
   step: 'Driver Step',
   datasetId: 'Dataset ID',
   requestTrace: 'Request Trace',
-  sessionToken: 'Session Token',
   sessionTrace: 'Session Trace',
-  stringifiedStack: 'Stack Trace',
   utc: 'UTC Time',
-  version: 'Version'
+  version: 'Version',
+  uuid: 'Rollbar UUID'
 };
 
 export function addInfoToError(err, props) {
-  let debug = [];
+  let debug = []; 
   _.forOwn(props, (v, k) => {
-    if (!_.isEmpty(v) && v !== err.message) {
+    if (!_.isEmpty(v) && v !== err.message && 
+        k !== 'utc' && 
+        k!== 'code' && 
```

Comment:
nitpick, but spacing is off here
```suggestion
        k !== 'code' && 
```